CSharp/Microsoft.Bot.Builder.Teams/Middlewares/TeamsTenantFilteringMiddleware.cs

```diff
@@ -51,23 +56,28 @@ public TeamsTenantFilteringMiddleware(IEnumerable<string> allowedTenantIds)
         /// </remarks>
         /// <seealso cref="ITurnContext" />
         /// <seealso cref="Schema.IActivity" />
-        public Task OnTurnAsync(ITurnContext turnContext, NextDelegate next, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task OnTurnAsync(ITurnContext turnContext, NextDelegate next, CancellationToken cancellationToken = default(CancellationToken))
         {
-            // Ignoring cases where ChannelData is missing or does not contain TenantId.
-            if (turnContext.Activity.ChannelData != null)
+            if (!turnContext.Activity.ChannelId.Equals("msteams", StringComparison.OrdinalIgnoreCase))
```

Comment:
Would be nicer to use the constant [`Connector.Channels.Msteams`](https://github.com/Microsoft/botbuilder-dotnet/blob/c3297158bc9ff8148c9e221719b3ffe564a82227/libraries/Microsoft.Bot.Connector/Channels.cs)?

Reply:
Changed to use this but now I also needed to update all the dependencies to latest bot builder =O